.tekton/windows-machine-config-operator-bundle-master-pull-request.yaml

```diff
@@ -248,8 +248,6 @@ spec:
       params:
       - name: BINARY_IMAGE
         value: $(params.output-image)
-      - name: BASE_IMAGES
```

Comment:
@sebsoto  please include a link to the Konflux upgrade docs in the description of the PR